src/os/bluestore/KernelDevice.cc

```diff
@@ -712,9 +716,11 @@ int KernelDevice::aio_read(
     ++ioc->num_pending;
     aio_t& aio = ioc->pending_aios.back();
     aio.pread(off, len);
-    for (unsigned i=0; i<aio.iov.size(); ++i) {
-      dout(30) << "aio " << i << " " << aio.iov[i].iov_base
-	       << " " << aio.iov[i].iov_len << dendl;
+    if (cct->_conf->subsys.should_gather(ceph_subsys_bdev, DEBUG_LEVEL)) {
```

Comment:
@majianpeng @ifed01 probably #18873 is better alternative to address this issue?